keps/sig-scheduling/3094-pod-topology-spread-considering-taints/kep.yaml

```diff
@@ -0,0 +1,49 @@
+title: Take taints/tolerations into consideration when calculating PodTopologySpread skew
+kep-number: 3094
+authors:
+  - "@kerthcet"
+owning-sig: sig-scheduling
+participating-sigs:
+status: implementable
+creation-date: 2021-12-30
+reviewers:
+  - "@ahg-g"
+  - "@alculquicondor"
+  - "@Huang-Wei"
+approvers:
+  - TBD
+
+see-also:
+  - "/keps/sig-scheduling/895-pod-topology-spread"
+  - "/keps/sig-scheduling/1258-default-pod-topology-spread"
+
+##### WARNING !!! ######
+# prr-approvers has been moved to its own location
+# You should create your own in keps/prod-readiness
+# Please make a copy of keps/prod-readiness/template/nnnn.yaml
+# to keps/prod-readiness/sig-xxxxx/00000.yaml (replace with kep number)
+#prr-approvers:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.24"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.24"
+  beta: "v1.25"
+  stable: "v1.26"
```

Comment:
generally 2 milestones are needed for stable. But you can remove this field for now, I think.